packages/integrations/image/src/index.ts

```diff
@@ -49,16 +49,15 @@ export default function integration(options: IntegrationOptions = {}): AstroInte
 	function getViteConfiguration() {
 		return {
 			plugins: [createPlugin(_config, resolvedOptions)],
-			optimizeDeps: {
-				include: ['image-size'].filter(Boolean),
-			},
```

Comment:
It doesn't look like `image-size` is being accessed in the client through some testing, and Vite/Astro was reporting that it can't resolve `image-size` with pnpm, so I removed this.